aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -621,9 +621,10 @@ at::Tensor PackedLinearWeightsQnnp::apply_relu(
 #endif // USE_PYTORCH_QNNPACK
 
 #if AT_MKLDNN_ENABLED()
-template <bool ReluFused>
+template <PostOps post_op>
 at::Tensor PackedLinearWeightsOnednn::apply_impl(
     at::Tensor input,
+    double negative_slope,
```

Comment:
Suggest not to add the post-op specific parameter into the args of a general function. Consider to do something like below:
https://github.com/pytorch/pytorch/pull/86583/files#diff-c24c006375f92e12dd5b062f143b62d41f347477b0efb7c25e3824ac41f0ad7eR184